homeassistant/components/myq/__init__.py

```diff
@@ -63,3 +73,42 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry):
         hass.data[DOMAIN].pop(entry.entry_id)
 
     return unload_ok
+
+
+class MyQEntity(CoordinatorEntity):
+    """Base class for MyQ Entities."""
+
+    def __init__(self, coordinator: DataUpdateCoordinator, device) -> None:
```

Comment:
```suggestion
    def __init__(self, coordinator: DataUpdateCoordinator, device) -> None:
```
Add the typing for the device here as well